plugins/targetlocker/dblocker/dblocker.go

```diff
@@ -240,11 +275,8 @@ func (d *DBLocker) Lock(ctx xcontext.Context, jobID types.JobID, duration time.D
 	if err := validateTargets(targets); err != nil {
 		return fmt.Errorf("invalid lock request: %w", err)
 	}
-	ctx.Debugf("Requested to lock %d targets for job ID %d: %v", len(targets), jobID, targets)
-	if len(targets) == 0 {
-		return nil
-	}
-	_, err := d.handleLock(ctx, int64(jobID), targetIDList(targets), uint(len(targets)), duration, false)
+	_, err := d.handleLock(ctx, int64(jobID), targetIDList(targets), uint(len(targets)), duration, false /* requireLocked */, false /* allowConflicts */)
+	ctx.Debugf("Lock %d targets for job %d for %s: %v", len(targets), jobID, duration, err)
```

Comment:
we can remove the jobID now, it is in the contextual logger (but can do in another PR)

Reply:
done